src/mainwindow.cpp

```diff
@@ -1245,8 +1228,8 @@ void MainWindow::FindNextH()
 
 void MainWindow::FindImpl(int offset, bool findHighlight)
 {
-    QTreeView * tree = GetCurrentTreeView();;
-    TreeModel * model = GetTreeModel(tree);
+    QTreeView * tree = GetCurrentTreeView();
```

Comment:
I think you can delete this line, variable tree is not used

Reply:
tree is used in the code below, e.g. in line 1242
`int start = tree->currentIndex().row();`

Reply:
you are right, I didn't see that